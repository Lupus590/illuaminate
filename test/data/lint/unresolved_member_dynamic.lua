-- config: (lint (only var:unresolved-member) (dynamic-modules a_module))

local a_module, b_module = require("a_module"), require("b_module")
print(a_module.unknown)
print(b_module.unknown)
