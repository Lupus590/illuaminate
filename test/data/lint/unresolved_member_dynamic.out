unresolved_member_dynamic.lua:[5:7-5:22]: Unknown field unknown in b_module [var:unresolved-member]
   │
 5 │ print(b_module.unknown)
   │       ^^^^^^^^^^^^^^^^
No errors and 1 warnings
